doc/source/whatsnew/v1.1.0.rst

```diff
@@ -333,6 +333,7 @@ Other enhancements
 - :meth:`read_csv` now accepts string values like "0", "0.0", "1", "1.0" as convertible to the nullable boolean dtype (:issue:`34859`)
 - :class:`pandas.core.window.ExponentialMovingWindow` now supports a ``times`` argument that allows ``mean`` to be calculated with observations spaced by the timestamps in ``times`` (:issue:`34839`)
 - :meth:`DataFrame.agg` and :meth:`Series.agg` now accept named aggregation for renaming the output columns/indexes. (:issue:`26513`)
+- :meth:`DataFrame.idxmax`, :meth:`DataFrame.idxmin`, :meth:`Series.idxmax`, :meth:`Series.idxmax`, :meth:`Series.argmax` and :meth:`Series.argmin` all now have new keep parameter. The functionality of this parameter is similar to the keep parameter found in the nlargest function. First prioritizes returns first occurrence of value in case of ties, last similarly returns the last, and all returns all occurrences of the value. (:issue:`34205`)
```

Comment:
all now have gained a new 'keep' parameter (use backticks around keep). you can also add a reference to Series.nlargest